index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>TuTangle</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <div class="headerTab">
+          <div class="TuTangle">
+            TuTangle
+          </div>
+        </div>
```

Comment:
if it's possible, can you use just one **div** here?
and can you change the second div to `h1`. I think it makes more sense that way.